packages/atomic/src/utils/initialization-utils.tsx

```diff
@@ -220,7 +220,7 @@ export function BindStateToController(
   };
 }
 
-export type I18nState = Record<string, () => string>;
+export type I18nState = Record<string, (variables?: Object) => string>;
```

Comment:
Is `Object` the better type here ? Isn't there a specific type that `18n` declares ? 

I'm seeing something called `TFunction` when I grep their codebase quickly.

```
export interface i18n {
  // Expose parameterized t in the i18next interface hierarchy
  t: TFunction;
  

   [....]
}
```